docs/types/value-types.rst

```diff
@@ -65,10 +65,10 @@ This means that, for example ``~int256(0) == int256(-1)``.
 Shifts
 ^^^^^^
 
-The result of a shift operation has the type of the left operand, truncating the result to match the type.
+The result of a shift operation has the type of the left operand, truncating the result to match the type. This means an "overflow" will not be "checked".
```

Comment:
Truncating the result pretty much means that there will be no overflow checks - so in that sense I would actually say that this *was* documented. But it for sure doesn't hurt to be absolutely clear and explicit.
To me the quoting seems a bit off, though. What about
```suggestion
This means that contrary to checked arithmetic, if the result of a bit shift operation will not fit its type, the transaction will *not* revert.
```

Reply:
Good wording, thank you

Reply:
Sorry, apparently I mistakenly erased the first sentence in my suggestion, so now it vanished - we'll want to keep that.